test/poll/async/get.test.js

```diff
@@ -31,8 +28,8 @@ test('pull.async.get', t => {
         { author: piet, position: { poll, choice: 2, reason: 'scuttles 4life' } }
       ]),
       pull.asyncMap((t, cb) => {
-        ChooseOne(t.position, (err, position) => {
-          t.position = position
+        ChooseOne(t.author)(t.position, (err, position) => {
```

Comment:
there's a bug here where `t.author` is a server ... but doesn't have `get` method on it .. putting `server` in here I don't think worked either.